release_build_files/Android/firebase_dependencies.gradle

```diff
@@ -27,8 +27,7 @@ def firebaseDependenciesMap = [
   'firestore' : ['com.google.firebase:firebase-firestore:24.1.2'],
   'functions' : ['com.google.firebase:firebase-functions:20.1.0'],
   'gma' : ['com.google.android.gms:play-services-ads:20.6.0',
-           'com.google.firebase:firebase-analytics:21.0.0',
-           'com.google.android.gms:play-services-base:18.0.1'],
```

Comment:
does this mean GMA doesn't need dependency on gms play services?

Reply:
[Jon reworked the dependencies in a PR](https://github.com/firebase/firebase-cpp-sdk/commit/6985af993aa71e3accbd8f80f715991001478a56) that already did something similar for AdMob and Analytics.  Just merging in his fix and reflecting it in the new AdMob (aka GMA) to match.